react-live.d.ts

```diff
@@ -0,0 +1,30 @@
+export as namespace ReactLive;
+
+import { Component } from 'react'
+
+export interface LiveProviderProps {
+  code: string;
+  error?: string;
+  onError?: Function;
+  onChange?: Function;
+  element?: any;
+}
+
+export class LiveProvider extends Component<LiveProviderProps, {}>{}
+
+export interface EditorProps extends HTMLElement {
+  onChange: Function;
+  onKeyDown: Function;
+  onKeyUp: Function;
+  onClick: Function;
+  code: string;
+}
+
```

Comment:
I think the normal `Editor` is missing :) https://github.com/FormidableLabs/react-live/blob/master/src/index.js#L13